lib/test/t_SmolyakExperiment_std.cxx

```diff
@@ -417,27 +416,6 @@ void test_7()
   assert_equal(dimensionBis, 3);
   assert_equal(weightDimensionBis, 83);
   ResourceMap::SetAsBool("SmolyakExperiment-MergeQuadrature", true);
-  // Test 3 : Set tolerances to zero
-  Scalar defaultRelativeEpsilon(ResourceMap::GetAsScalar("SmolyakExperiment-MergeRelativeEpsilon"));
```

Comment:
I agree with @mbaudin47 that removing a large part of the test is problematic. For the time being, @pgrt manually removes this part of the test for problematic architectures. Couldn'tr we make this part of the test architecture-dependent?